### ISSUE TITLE
Using redux forms

### ISSUE THREAD

**Issue by bbrock25** [spartansystems/rn-booster-kit#5]
We need to define a some standard patterns for how we want to interact with redux forms:
- form validation, local and server side
- error message handling
- other things...

**Comment by jasontwong** [comment]
Yeah. The `react-native` integration is pretty different so we'll probably have to run some tests to see what would be best.
